src/views/product-tutorials-view/components/sitemap/helpers.ts

```diff
@@ -8,9 +8,11 @@ export function formatSitemapCollection(
 	collection: ClientCollection
 ): SitemapCollection {
 	return {
+		id: collection.id,
 		slug: collection.slug,
 		name: collection.name,
 		tutorials: collection.tutorials.map((t: ClientTutorialLite) => ({
+			id: t.id,
```

Comment:
Side quest, `id` were missing here, they're used as `key` values, have added them in.